src/status_im/contexts/wallet/events.cljs

```diff
@@ -204,18 +204,23 @@
 
 (rf/reg-event-fx :wallet/get-wallet-token-for-all-accounts
  (fn [{:keys [db]}]
-   {:fx (->> (get-in db [:wallet :accounts])
-             vals
-             (mapv
-              (fn [{:keys [address]}]
-                [:dispatch [:wallet/get-wallet-token-for-account address]])))}))
+   (let [force-balance-refresh? (boolean (get-in db [:wallet :force-balance-refresh?]))]
+     {:db (update-in db [:wallet] dissoc :force-balance-refresh?)
```

Comment:
The name of key in re-frame db should describe a current state of the app. So I'd suggest renaming `:force-balance-refresh?`->`:balance-should-be-refetched?`. Because it sounds more like a state description than like an argument name. Also using `fetch` instead of `refresh` is more inline with underlying status-go function name `FetchOrGetCachedWalletBalances`.
Wdyt?